x/batch/decorator_test.go

```diff
@@ -296,9 +278,7 @@ func TestDecoratorNew(t *testing.T) {
 					}
 				}
 
-				if spec.err != nil {
-					assert.Equal(t, true, spec.err.Is(err))
-				}
+				assert.Equal(t, true, spec.err.Is(err))
```

Comment:
It would be much better to print what error was given instead. Now it fails and you cannot know why.

Reply:
done!